anxcloud/struct_dns_zones.go

```diff
@@ -1,6 +1,9 @@
 package anxcloud
 
-import "go.anx.io/go-anxcloud/pkg/clouddns/zone"
+import (
+	clouddns "go.anx.io/go-anxcloud/pkg/apis/clouddns/v1"
```

Comment:
when touching this file already and adding generic client usage, would it be viable to replace the old-style client in this file?